main.py

```diff
@@ -38,9 +39,25 @@ def main(self): # The main function contains the most important code
         time.sleep(2) # Wait a few seconds
         self.slowType("Made by: Drillenissen#4268 && Benz#4947", .02) # Print who developed the code
         time.sleep(1) # Wait a little more
-        self.slowType("\nInput How Many Codes to Generate and Check: ", .02, newLine = False) # Print the first question
+        self.slowType("\nRun until valid code is found? y/n ", .02, newLine = False)
+        infinite_choice = input('')
```

Comment:
`elif infinite_choice == "n" or infinite_choice == "N":`
`if infinite_choice == "y" or infinite_choice == "Y":`

Those "or" in the beginning are unnecessary.
You could just use:

`infinite_choice = input('').lower()`

and then all of those wouldn't be necessary.

Reply:
You could add a `.strip()` to infinite choice too.
`infinite_choice = input('').lower().strip()`
making the user life easier.

Reply:
Or just use a check if it is in like `if "y" in input.lower()`